src/CMakeLists.txt

```diff
@@ -68,12 +68,6 @@ set(LIME_SUITE_SOURCES
     include/limesuite/SDRDevice.cpp
 )
 
-SET_SOURCE_FILES_PROPERTIES( 
```

Comment:
these files are originally intended to be compilable with C only for microcontrollers, so they should not be changed into C++ or use C++ features without conditional defines.